keycloak/testcontainers/keycloak/__init__.py

```diff
@@ -33,45 +32,33 @@ class KeycloakContainer(DockerContainer):
             >>> with KeycloakContainer() as kc:
             ...    keycloak = kc.get_client()
     """
-    def __init__(self, image="jboss/keycloak:latest", username: Optional[str] = None,
+    def __init__(self, image="quay.io/keycloak/keycloak:latest", username: Optional[str] = None,
                  password: Optional[str] = None, port: int = 8080) -> None:
         super(KeycloakContainer, self).__init__(image=image)
-        self.username = username or os.environ.get("KEYCLOAK_USER", "test")
-        self.password = password or os.environ.get("KEYCLOAK_PASSWORD", "test")
+        self.username = username or os.environ.get("KEYCLOAK_ADMIN", "test")
+        self.password = password or os.environ.get("KEYCLOAK_ADMIN_PASSWORD", "test")
         self.port = port
         self.with_exposed_ports(self.port)
 
     def _configure(self) -> None:
-        self.with_env("KEYCLOAK_USER", self.username)
-        self.with_env("KEYCLOAK_PASSWORD", self.password)
+        self.with_env("KEYCLOAK_ADMIN", self.username)
+        self.with_env("KEYCLOAK_ADMIN_PASSWORD", self.password)
+        self.with_command("start-dev")
 
     def get_url(self) -> str:
         host = self.get_container_host_ip()
         port = self.get_exposed_port(self.port)
         return f"http://{host}:{port}"
 
-    @wait_container_is_ready(requests.exceptions.ConnectionError, requests.exceptions.ReadTimeout)
-    def _connect(self) -> None:
-        url = self.get_url()
-        response = requests.get(f"{url}/auth", timeout=1)
-        response.raise_for_status()
-
     def start(self) -> "KeycloakContainer":
         self._configure()
-        super().start()
-        self._connect()
+        container = super().start()
+        wait_for_logs(container, 'Listening on:')
```

Comment:
I haven't tried this yet, but by passing `KC_HEALTH_ENABLED=true` as a environment variable I believe it might be possible to keep the `self._connect()` if instead of pooling `{url}/auth` we pool `{url}/health`.